.github/workflows/integration-sqlite.yml

```
@@ -0,0 +1,169 @@
+
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+      openldap:
+        image: ghcr.io/nextcloud/continuous-integration-openldap:openldap-7
+        env:
+          SLAPD_DOMAIN: nextcloud.ci
+          SLAPD_ORGANIZATION: Nextcloud
+          SLAPD_PASSWORD: admin
+          SLAPD_ADDITIONAL_MODULES: memberof
+
+    steps:
+      - name: Checkout server
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          submodules: true
+
+      - name: Checkout Talk app
+        if: ${{ matrix.test-suite == 'videoverification_features' }}
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          repository: nextcloud/spreed
+          path: apps/spreed
+          ref: ${{ matrix.spreed-versions }}
+
+      - name: Set up php ${{ matrix.php-versions }}
+        uses: shivammathur/setup-php@c5fc0d8281aba02c7fda07d3a70cc5371548067d # v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          # https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation
+          extensions: bz2, ctype, curl, dom, fileinfo, gd, iconv, imagick, intl, json, ldap, libxml, mbstring, openssl, pcntl, posix, redis, session, simplexml, xmlreader, xmlwriter, zip, zlib, sqlite, pdo_sqlite
+          coverage: none
+          ini-file: development
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up production dependencies
+        run: composer i --no-dev
+
+      - name: Set up behat dependencies
+        working-directory: build/integration
+        run: composer i
+
+      - name: Set up Talk dependencies
+        if: ${{ matrix.test-suite == 'videoverification_features' }}
+        working-directory: apps/spreed
+        run: composer i --no-dev
+
+      - name: Set up Nextcloud
+        env:
+          DB_PORT: 4444
+        run: |
+          mkdir data
+          ./occ maintenance:install --verbose --database=sqlite --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
+          ./occ config:system:set hashing_default_password --value=true --type=boolean
+
+      - name: Configure caching
+        if: ${{ matrix.test-suite == 'ldap_features' }}
+        run: |
+          ./occ config:system:set redis host --value=localhost
+          ./occ config:system:set redis port --value=6379 --type=integer
+          ./occ config:system:set redis timeout --value=0 --type=integer
+          ./occ config:system:set memcache.local --value='\OC\Memcache\Redis'
+          ./occ config:system:set memcache.distributed --value='\OC\Memcache\Redis'
```

Why not use redis everywhere might make tests faster?
We could then simplify to:
```suggestion
          cp tests/redis.config.php config/
          ./occ maintenance:install --verbose --database=sqlite --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
          ./occ config:system:set hashing_default_password --value=true --type=boolean
```

---

(oh this only works with the changes from the unit test PR as there I changed the redis config to work with GH actions)

---

> Why not use redis everywhere might make tests faster?

Because I'm not sure we should have this. CI is not only about speed. But e.g. when redis is there, behaviour changes (e.g. BFP has a redis and a DB backend, etc), so each test should decide if they need it.